yarn.lock

```diff
@@ -3788,6 +4266,17 @@ browserslist@^1.3.6, browserslist@^1.5.2, browserslist@^1.7.6:
     caniuse-db "^1.0.30000639"
     electron-to-chromium "^1.2.7"
 
+browserslist@^4.17.5, browserslist@^4.19.1:
```

Comment:
this should not be in the PR

Reply:
I don't know how to split it from the rest at this point. Shouldn't browserslist-config-cozy takes precedence anyway?

Reply:
then, why this deps has changed, if browserslist-config-cozy didn't get updated?

Reply:
@trollepierre I don't know anymore, there have been multiple dependencies rebasing.

Reply:
ok, let's go for it!